### PR TITLE
close #114 create a simple product instead of variation when variations == 1

### LINKED ISSUE #114: Fail to create product if variation has the same SKU of parent product
### Steps:

1. Import any products with standard size, and variation SKU is the same of product SKU

**Expected**: Product to be created

**Result**: Only variation is there with type product variant without parent product

### Why?
Into the import process, parent product created, then while adding child product, this function getProductByIdOrSKU() get the parent and update it to variation instead of adding it again as a child.

### Proposed solution:
Instead of creating variable product with one variant, create it as simple product and get the price, qty ... etc from the variation to the parent

### Test checklist
- [ ] Validate normal and one variant products creation is working as before
- [ ] Create sales orders for both orders type, make sure order arrived to Knawat as expected
- [ ] Test the plugin on production website to validate the backward compatibility, expect to update all old products from product_variation to simple product

### PR DESCRIPTION
**Pull request by hamusthekiller**
I handled a case where the plugin should create a simple product instead of a variation product when the number of variations is equal to 1 and parent sku = child sku.
I did a simple migration task for old versions of the plugin to fix this issue.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -59,6 +59,9 @@ Knawat WooCommerce DropShipping allows you to Add products to your WooCommerce s
 
 == Changelog ==
 
+= 2.0.7 =
+* FIX: Product with only 1 variation gets added as a simple product instead of variable.
```
Comment (ElGreetly) [inline README.txt]: what is the variable the products was variable?

File: dropshipping-woocommerce.php [hunk 2]
```diff
@@ -21,6 +21,20 @@
 
 if( ! class_exists( 'Knawat_Dropshipping_Woocommerce' ) ):
 
+	//IMPORTANT in order for the migration task to work
+	global $wpdb;
+	$plugin_data = get_file_data(__FILE__, array('Version' => 'Version'), false);
+	$plugin_version = $plugin_data['Version'];
+	$wpdb->replace($wpdb->postmeta, array('meta_id' => '1000000', 'meta_key' => 'knawat-old-version', 'meta_value' => '2.0.6'));
+	$wpdb->replace($wpdb->postmeta, array('meta_id' => '1000001', 'meta_key' => 'knawat-current-version', 'meta_value' => $plugin_version));
```
Comment (ElGreetly) [inline dropshipping-woocommerce.php]: These lines should be dynamic, not static!

File: README.txt [hunk 3]
```diff
@@ -4,7 +4,7 @@ Tags: dropshipping, woocommerce, woocommerce dropshipping, dropship, woo dropshi
 Requires PHP: 5.3
 Requires at least: 4.0
 Tested up to: 5.3
```
Comment (HazemKhaled) [inline README.txt]: change to latest wordpress version

Comment (HazemKhaled) [reply README.txt]: and woocommerce version

**Pull request merged**